### ISSUE TITLE
ACF rules: warn if team gets overtime tossups and TUH==20

### ISSUE THREAD

**Issue by rjaguar3** [mostafab/neg5#41]
In untimed 20/20 formats, TUH==20 if and only if overtime is not played.  If overtime is played, TUH should be greater than 20.  (The converse is not true, as a game can go into overtime without a team getting an overtime tossup if the game ends by a neg.)

**Comment by rjaguar3** [comment]
This only applies in formats where every non-overtime tossup leads to a bonus.

**Comment by jonahgreenthal** [comment]
I don't understand the statement about the converse. If a team negs an overtime tossup, it still counts as a TUH.

**Comment by rjaguar3** [comment]
Yes, but neither side should get credit for overtime tossup (tossup-without-bonus) in its stats.

**Comment by jonahgreenthal** [comment]
That's true; I didn't realize you were using "overtime is played" to mean "tossups without bonuses is positive on at least one side" (which is at best an abuse of terminology).

**Comment by rjaguar3** [comment]
Neg5 uses "Overtime TU" on the stat entry screen (which is an accurate description for the major formats).

**Comment by jonahgreenthal** [comment]
New bug report: label that better :-)
